docs/REST API/rest-api.md

```diff
@@ -826,6 +826,9 @@ None.
     "enable_sso_idp_login": false,
     "enable_jit_provisioning": false
   },
+  "conditional_access": {
+    "microsoft_tenant_id": "<TENANT ID>"
+  },
```

Comment:
Tenant ID is not saved until Fleet server receives confirmation from the fleetdm.com proxy.

If this exists, we know that conditional access has been configured.